apps/api/src/routes/v1_keys_verifyKey.ts

```diff
@@ -310,6 +323,7 @@ export const registerV1KeysVerifyKey = (app: App) =>
       permissionQuery: req.authorization?.permissions,
       ratelimit: req.ratelimit,
       ratelimits: req.ratelimits,
+      remaining: req.remaining ?? { cost: 1 },
```

Comment:
If you are feeling fancy let's use the same number 😄

Reply:
actually good catch, this should never be optional here and we can remove the fallback :)